library/src/main/java/com/airbnb/android/airmapview/WebAirMapViewBuilder.java

```diff
@@ -1,11 +1,15 @@
 package com.airbnb.android.airmapview;
 
+import android.text.TextUtils;
+
 /**
  * AirMapView map that uses the web based Google Maps implementation.
  */
 public class WebAirMapViewBuilder implements AirMapViewBuilder<WebViewMapFragment, AirMapType> {
 
   private AirMapType options;
+  private String mapboxMapId;
+  private String mapboxAccessToken;
```

Comment:
these should be moved to `MapboxWebViewMapFragment`, and remove the setters for these and instead extract them from the `AirMapType`